### PR TITLE
[VideoMAE] Improve code examples

### PR DESCRIPTION
**Pull request by NielsRogge**
# What does this PR do?

This PR simplifies the code examples of VideoMAE, and adds a seed to make sure the video classifier always predicts "eating spaghetti" on the video (as, due to the sampling of frames, it may occur the model predicts another class, like "eating ice cream"):

```
1019 
1020         >>> inputs = feature_extractor(list(video), return_tensors="pt")
1021 
1022         >>> with torch.no_grad():
1023         ...     outputs = model(**inputs)
1024         ...     logits = outputs.logits
1025 
1026         >>> # model predicts one of the 400 Kinetics-400 classes
1027         >>> predicted_label = logits.argmax(-1).item()
1028         >>> print(model.config.id2label[predicted_label])
Expected:
    eating spaghetti
Got:
    eating ice cream
```

Weirdly, this wasn't caught by the doc test CI. It could have to do with the addition of i`mport numpy as np` to the code snippet.

### CODE REVIEW

**Review by ydshieh — APPROVED** [review]
LGTM, thanks @NielsRogge .

I don't know why, but so far I always get `spaghetti`, despite the random indices (I removed the seed)

**Review by LysandreJik — APPROVED** [review]
Clean!
